airflow/sensors/external_task.py

```diff
@@ -134,20 +146,23 @@ def __init__(
         self.execution_delta = execution_delta
         self.execution_date_fn = execution_date_fn
         self.external_dag_id = external_dag_id
+        self.external_task_group_id = external_task_group_id
         self.external_task_id = external_task_id
         self.check_existence = check_existence
```

Comment:
This is `False` by default, which may make sense for external_dag or external_task. But external_task_group has to check and get an existing dag in order to get the list of task_ids. 

I wonder if we can change the default to True or even have `check_existence` enabled required? This can give more useful errors when the external task/dag does not exist. Also, I can't think of the reason to have a Sensor waiting for an object that doesn't exist until it times out